src/pages/Home.tsx

```diff
@@ -43,7 +43,7 @@ export const Home: React.FC = () => {
                 navigate('/workspace');
             })
             .catch((error) => console.error(error));
-    }, [navigate]);
+    }, [globalDispatch, navigate, workspaceDispatch]);
```

Comment:
Let's keep the dependency array in the same order as the functions are called.
```suggestion
    }, [workspaceDispatch, globalDispatch, navigate]);
```

Reply:
I was keeping them in the alphabetical order btw. Just preference.